public/js/services/Api.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import moment from 'moment';
+
+export const httpClient = axios.create({
+    // baseURL: 'https://productionmetrics.local.dev-gutools.co.uk/', // Use this url in dev. TODO: Add proper ENV management
```

Comment:
We could handle this by sending a client config object through to frontend. This is something the targeting app does https://github.com/guardian/targeting/blob/master/app/controllers/Application.scala#L19 but I think this can be left for another PR